package.json

```diff
@@ -167,7 +167,7 @@
     "@gnosis.pm/safe-apps-sdk": "4.3.0-next.2",
     "@gnosis.pm/safe-apps-sdk-v1": "npm:@gnosis.pm/safe-apps-sdk@0.4.2",
     "@gnosis.pm/safe-core-sdk": "^0.3.1",
-    "@metissafe/safe-deployments": "^1.5.0",
+    "@gnosis.pm/safe-deployments": "^1.7.0",
```

Comment:
@titandac 
Let's use "@metissafe/safe-deployments": "^1.5.1", I forgot to push changes yesterday, we need to use the old testnet contracts for now, we can change this later.